v3/skme/ZenoErgo/Zeno004.json

```diff
@@ -0,0 +1,331 @@
+{
+    "name": "SKME Zeno 60% Ergo",
+    "vendorId": "0x4048",
+    "productId": "0x0001",
+    "matrix": {
+      "rows": 5,
+      "cols": 15
+    },
+    "layouts": {
+      "labels": [
+        "Full Backspace",
+        "Full Right Shift",
+        "Long Left Space (2.75u)",
+        "Short Right Space (2u)",
+        "1.25u Bottom Row"
+      ],
+      "keymap": [
+  {
+    "name": "zenokle"
```

Comment:
Please remove this

Reply:
how did this get past validation script? @ValdisThomann

Reply:
My bad, this is removed.